digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -175,6 +175,20 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
           .map(midPoint)
           .getOrElse(Map("success" -> false, "reason" -> ("ID:" + id + " not found")))
     }
+
+  }
+
+  get("/roadlinks/project/prefillfromvvh") {
```

Comment:
duplicate entry point

Reply:
Already removed it mate, thank you though.